order/buyer/receive/simple-order-event.md

```diff
@@ -0,0 +1,167 @@
+---
+description: How to receive a simple order response sent by the supplier
+---
+
+# Receive an order response
+
+Tradecloud will send a purchase order response to the buyer when an order event has been triggered.
+
+## `simpleOrderEvent`
+
+This page assumes you are using the webhook with the simple delivery schedule in the `simpleOrderEvent`.
+
+* `orderId`: the Tradecloud order identifier
+* `buyerOrder`: the buyer part of the order, see [Buyer order](#buyer-order)
+* `supplierOrder`: the supplier part of the order, see [Supplier order](#supplier-order)
+* `indicators.deliveryOverdue` is true when at least one order line is overdue.
+* `status.processStatus`: is the aggregate of all lines' [Process statuses](#process-status).
+* `status.logisticsStatus`: is the aggregate of all lines' [Logistics statuses](#logistics-status).
+* `version`: the Tradecloud order version number
+* `eventDates`: some key order event date/times
+* `meta`: meta information, including source and trace info, about this messsage
+* `lastUpdatedAt`: is the latest date time the order has been changed.
+
+### Buyer order
+
+`buyerOrder` is mostly an echo of your order fields as explained in [Issue a new order](../issue/#order-body-json-objects)
+
+* `supplierAccountNumber`: the supplier account number as known in your ERP system
+
+### Supplier order
+
+`supplierOrder` contains the supplier order fields:
+
+* `companyId`: the supplier's Tradecloud company identifier.
+* `buyerAccountNumber`: your account number as known in the supplier's ERP system.
+* `description`: a free format additional description of this order by the supplier.
+* `contact`: the supplier employee responsible for this order.
+* `properties`: are key-value based custom fields, added by the supplier.
+* `notes`: are simple custom fields, added by the supplier.
+* `documents`: contain meta data and link of attached documents by the supplier.
+
+{% page-ref page="download-document.md" %}
+
+### Status
```

Comment:
Don't we have this status explanation already somewhere else? It feels like we should refer to a "central" explanation here.

Reply:
Yes, in `receive/README.md` for example, but I think it is more convenient for the user to have it in the same page.